routes/web.php

```diff
@@ -7,3 +7,7 @@
     return view('welcome');
 });
 Route::get('/contact', 'App\Http\Controllers\contacts\Contacts@index');
+Route::post('/contact', 'App\Http\Controllers\contacts\Contacts@post');
+Route::get('/contact/confirm', 'App\Http\Controllers\contacts\Contacts@confirm_index');
+Route::post('/contact/confirm', 'App\Http\Controllers\contacts\Contacts@send_mail');
+Route::get('/contact/complete', 'App\Http\Controllers\contacts\Contacts@complete_index');
```

Comment:
```suggestion
Route::prefix('contact')->group(function () {
    Route::get('/', 'App\Http\Controllers\contacts\Contacts@index')->name("contactInputGet");
    Route::post('/', 'App\Http\Controllers\contacts\Contacts@post')->name("contactInputPost");
    Route::get('confirm', 'App\Http\Controllers\contacts\Contacts@confirm_index')->name("confirmIndex");
    Route::post('confirm', 'App\Http\Controllers\contacts\Contacts@send_mail')->name("contactSendmail");
    Route::get('complete', 'App\Http\Controllers\contacts\Contacts@complete_index')->name("completeIndex");
});
```
この書き方で書いておくとURLを変更しないといけない時にすごく便利。